pkg/bootstrap/docker/openshift/helper.go

```diff
@@ -730,6 +730,23 @@ func (h *Helper) updateConfig(configDir string, opt *StartOptions) error {
 		nodeCfg.DNSIP = ""
 	}
 	nodeCfg.DNSBindAddress = ""
+
+	if h.supportsEnforceNodeAllocatableArgument() {
+		// Set the cgroup driver from the current docker
+		cgroupDriver, err := h.dockerHelper.CgroupDriver()
+		if err != nil {
+			return err
+		}
+		if nodeCfg.KubeletArguments == nil {
+			nodeCfg.KubeletArguments = configapi.ExtendedArguments{}
+		}
+		nodeCfg.KubeletArguments["cgroup-driver"] = []string{cgroupDriver}
+		if cgroupDriver != "systemd" {
```

Comment:
this should not be necessary with https://github.com/kubernetes/kubernetes/pull/45515

can you pull that locally and verify it works for you?

if you verify, we can carry a patch in origin.